cleantalk.php

```diff
@@ -190,6 +190,11 @@ function apbct_alt_session__save__WP_AJAX()
             }
         }
     }
+
```

Comment:
Why? And Why in this branch?

Reply:
Done https://github.com/CleanTalk/wordpress-antispam/commit/ee566069f5ccd69afa71b7d5efa84ca42ed86dbb